sdk/src/azure/core/az_context.c

```diff
@@ -34,7 +34,7 @@ AZ_NODISCARD int64_t az_context_get_expiration(az_context const* context)
 // and return the corresponding value. Returns AZ_ERROR_ITEM_NOT_FOUND is there are no nodes
 // matching the specified key.
 AZ_NODISCARD az_result
-az_context_get_value(az_context const* context, void const* key, void const** out_value)
+az_context_get_value(az_context const* context, void const* const key, void const** out_value)
```

Comment:
No, I don't want to change the source/implementation detail to jump through const hoops :)

Such implementation detail changes can be made separately, if the area owner cares to do so.

I only marked things that are known to be const if they made sense, and the implementation allowed it, given this is an implementation-only annotation.

Reply:
Ok, that makes sense.  Updating the implementation (and resulting signature) might be something to consider for the future for consistency in approach, since we are using a local copy of the pointer in `AZ_INLINE bool _az_log_classifications_are_valid(az_log_classification const* const classifications)`